jmix-gridexport/gridexport-flowui/src/main/java/io/jmix/gridexportflowui/GridExportProperties.java

```diff
@@ -30,6 +30,11 @@ public class GridExportProperties {
      */
     int exportAllBatchSize;
 
+    /**
+     * Choose appropriate pagination type to export data
```

Comment:
In configuration properties classes we start javadoc for fields with a noun. In your case there must be something like "Pagination type for all entities export".